system/Test/CIUnitTestCase.php

```diff
@@ -439,13 +439,13 @@ public function assertHeaderNotEmitted(string $header, bool $ignoreCase = false)
      * where the result is close but not exactly equal to the
      * expected time, for reasons beyond our control.
      *
-     * @param mixed $actual
+     * @param float|int $actual
      *
      * @throws Exception
      */
     public function assertCloseEnough(int $expected, $actual, string $message = '', int $tolerance = 1)
     {
-        $difference = abs($expected - (int) floor($actual));
+        $difference = abs($expected - (int) floor((float) $actual));
```

Comment:
This seems like it should be either/or. `floor()` supports `int` input so why the cast? unless the docblock type isn't certain?

Reply:
Yes, I removed `(float)`.